Makefile

```diff
@@ -46,7 +46,7 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 
 .PHONY: generate
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(CONTROLLER_GEN) object paths="./..."
```

Comment:
Why not create another special boilerplate go autogenerated.

Reply:
What for?

Reply:
copyright 😄

Reply:
Autogenerated files aren't copyrightable.

Reply:
can you put the boilerplate file empty, and keep this command as it is?

Reply:
I could but why would we keep an empty file around?

Reply:
> Autogenerated files aren't copyrightable.

I must disagree here. All files that are compiled to should have the license header even those that are auto-generated. It is not required that the year be kept up to date, so rather the validation that the year matches should be loosened